packages/services/api/src/modules/auth/lib/authz.ts

```diff
@@ -514,7 +520,7 @@ const actionDefinitions = {
 
 type Actions = keyof typeof actionDefinitions;
 
-type ActionStrings = Actions | '*';
+type ActionStrings = Actions | '*' | '*:describe';
```

Comment:
I listed this as a special action string in order to avoid having to get a major in typescript and further cooking the ide performance.

Reply:
Aside from `describe`, I see no other wildcard usage right now.